backend/pong/game.py

```diff
@@ -89,11 +89,13 @@ def calculate_ai_steps(self):
 
         return ball_y
 
-    def get_consts(self):
+    def get_consts(self, left_name: str, right_name: str) -> dict[str, any]:
         game_consts = {
             'players': {
                 'width': 2 * HALF_PLAYER_WIDTH,
                 'height': 2 * HALF_PLAYER_HEIGHT,
```

Comment:
we can do it like this, but to me makes more sense to use the selected display names and the frontend just gets them once when creating the play view.

Reply:
You can change it if you want, I just thought it would be a lot easier for the frontend that it get's the names from at the start of the match so that it doesn't require a separate request.